documentation/README.md

```diff
@@ -0,0 +1,9 @@
+# Developer Documentation
+
+If you are onboarding an entirely new service and starting from scratch please refer to the [new service onboarding documentation][new_service_docs]
+
+If you are looking to contribute to an existing package please refer to the [developer setup documentation][dev_setup_docs]
+
+<!-- LINKS -->
+[new_service_docs]: https://aka.ms
```

Comment:
Do these need updating?

Reply:
Yes but they won't exist until this PR is merged, at which point I will make a second PR to create these links